sharkfuser/build_tools/cmake/CTestMacros.cmake

```diff
@@ -129,7 +129,7 @@ function(add_sharkfuser_lit_test)
   )
 
   # Pass locations of tools in build directory to lit through `--path` arguments.
-  set(_LIT_PATH_ARGS)
+  set(_LIT_PATH_ARGS "--path" "$<TARGET_FILE_DIR:FileCheck>")
```

Comment:
Let's go back to what you had earlier and have all tools passed in explicitly to `add_sharkfuser_lit_test`:
```
add_sharkfuser_lit_test(
  SRC test_example_lit.cpp
  TOOLS 
    filecheck
    iree-opt
)
```